src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball ball1 = lottery.GetRandomBall();
+        Ball ball2 = lottery.GetRandomBall();
+        Ball ball3 = lottery.GetRandomBall();
```

Comment:
The method names should follow camelCase naming conventions. Instead of `GetRandomBall`, it should be `getRandomBall`.